### PR TITLE
adding hpo-toolkit package

### PR DESCRIPTION
**Pull request by iimpulse**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by iimpulse** [comment]
@conda-forge/help-python

File: recipes/hpo-toolkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python >=3.8
+    - networkx
+
+test:
+  imports:
+    - hpotk
+  requires:
+    - tox
+    - setuptools
```
Comment (BastianZim) [inline recipes/hpo-toolkit/meta.yaml:30]: If you only import, are these really required?

File: recipes/hpo-toolkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  summary: 'Simple & Efficient Hpo Toolkit'
+  license: GPL-3.0
+  license_file: LICENSE
+  description: |
+    hpo-toolkit is a simple and efficient library for mapping loinc test results
+    to hpo terms. Pure python, 3.8+
+
+extra:
+  recipe-maintainers:
+    - ielis
```
Comment (BastianZim) [inline recipes/hpo-toolkit/meta.yaml:45]: @ielis Are you happy to be a maintainer?

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/hpo-toolkit/meta.yaml (hunk 2 above)
Comment (ielis) [reply recipes/hpo-toolkit/meta.yaml:45]: Hi @BastianZim , yes, I am super happy to be a maintainer of the package! Thanks a lot!

**Pull request closed without merging**